plugins/python/src/modulewrap.cpp

```
@@ -61,32 +61,32 @@ using phlex::product_selector;
 // clang-format off
 struct phlex::experimental::py_phlex_module {
   PyObject_HEAD
-  phlex_module_t* ph_module;
+  phlex_module_t const* ph_module;
 };
 // clang-format on
 
-PyObject* phlex::experimental::wrap_module(phlex_module_t& module_)
+PyObject const* phlex::experimental::wrap_module(phlex_module_t const& module_)
```

Cppcheck (2.21.0)

[style] The function 'wrap_module' is never used.

(unusedFunction)